browser/src/map/handler/Map.SlideShow.js

```diff
@@ -190,6 +201,44 @@ L.Map.SlideShow = L.Handler.extend({
 		this._slideShow.contentWindow.focus();
 	},
 
+	_handleSlideWindowLoaded: function() {
+		const iframe = this._slideShowWindowProxy.document.querySelector('iframe');
+
+		if (iframe) {
+			iframe.contentWindow.focus();
+			iframe.contentWindow.addEventListener('keydown', this._onSlideWindowKeyPress.bind(this));
+		}
+	},
+
+	_generateSlideWindowHtml: function(title, slideURL) {
+		return `
+		<!DOCTYPE html>
+		<html lang="en">
+		<head>
+			<meta charset="UTF-8">
+			<meta name="viewport" content="width=device-width, initial-scale=1.0">
+			<title>${title}</title>
+			<style>
+				body, html {
+					margin: 0;
+					padding: 0;
+					height: 100%;
+					overflow: hidden; /* Prevent scrollbars */
+				}
+				iframe {
+					width: 100%;
+					height: 100%;
+					border: none;
+				}
+			</style>
+		</head>
+		<body>
+			<iframe src="${slideURL}"></iframe>
```

Comment:
The "url mask" works on chrome but not firefox which seems odd :

![image](https://github.com/CollaboraOnline/online/assets/65948705/f3888102-24be-4268-a4f4-991a8854126e)

Reply:
Firefox has automatic url navigation when we use `document.write(...)` and changes the url to the parent window. document.write may also cause additional problems (see https://developer.mozilla.org/en-US/docs/Web/API/Document/write) Solution is to use `document.documentElement.innerHTML = ...` changing only the innerHTML of the popup.